.github/workflows/releases.yml

```diff
@@ -58,92 +58,120 @@ jobs:
           echo "$( jq '.+{GatewayApiVersion:"${{ steps.get_release.outputs.tag_name }}"}' apps/GatewayApi/appsettings.json )" > src/GatewayAPI/appsettings.json
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@8c0edbc76e98fa90f69d9a2c020dcb50019dc325
-
-      - name: Docker build and push DataAggregator
-        uses: docker/build-push-action@c56af957549030174b10d6867f20e78cfd7debc5
+      - name: Publish Gateway Settings
+        uses: actions/upload-artifact@v3
         with:
-          file: ./apps/DataAggregator/Dockerfile
-          tags: |
-            ${{ steps.setup_tags.outputs.data_aggregator_tags }}
-          push: true
-          platforms: linux/amd64,linux/arm64
-          context: ./
-
-      - name: Docker build and push Gateway API
-        uses: docker/build-push-action@c56af957549030174b10d6867f20e78cfd7debc5
+          path: src/GatewayAPI/appsettings.json
+          name: gateway
+          retention-days: 1
+      - name: Publish DatabaseMigrations Settings
+        uses: actions/upload-artifact@v3
         with:
-          file: ./apps/GatewayApi/Dockerfile
-          tags: |
-            ${{ steps.setup_tags.outputs.gateway_api_tags }}
-          push: true
-          platforms: linux/amd64,linux/arm64
-          context: ./
-
-      - name: Docker build and push Database Migrations
-        uses: docker/build-push-action@c56af957549030174b10d6867f20e78cfd7debc5
+          path: src/DatabaseMigrations/appsettings.json
+          name: migrations
+          retention-days: 1
+      - name: Publish DataAggregator Settings
+        uses: actions/upload-artifact@v3
         with:
-          file: ./apps/DatabaseMigrations/Dockerfile
-          tags: |
-            ${{ steps.setup_tags.outputs.database_migrations_tags }}
-          push: true
-          platforms: linux/amd64,linux/arm64
-          context: ./
+          path: src/DataAggregator/appsettings.json
+          name: aggregator
+          retention-days: 1
+
+  docker-database-migrations-dockerhub:
+    name: (DatabaseMigrations) Docker
+    needs:
+      - setup-tags
+    uses: radixdlt/public-iac-resuable-artifacts/.github/workflows/docker-build.yml@main
+    with:
+      runs_on: ubuntu-22.04
+      image_registry: "docker.io"
+      image_organization: "radixdlt"
+      image_name: "babylon-ng-database-migrations"
+      tag: ${{ needs.setup-tags.outputs.database-migrations-tag }}
+      context: "."
+      dockerfile: "./apps/DatabaseMigrations/Dockerfile"
+      platforms: "linux/amd64"
+      environment: release
+      cache_tag_suffix: "release"
+      enable_dockerhub: "true"
+      restore_artifact: "true"
+      artifact_location: "src/DataAggregator/appsettings.json"
```

Comment:
May need to understand , Why do we need this. This file is available within context ?

Reply:
Check here: https://github.com/radixdlt/babylon-gateway/pull/191/files#r1131132885
Also the comment below to for more explanation

Reply:
And the cache_tag_suffix is entirely optional. Could also be the same cache image/tag as for the pr. I though about seperating them for making the cache invalidation more predictable.

Reply:
Ah okay. in that case , this should be database migration one
https://github.com/radixdlt/babylon-gateway/pull/191/files#diff-a62ff2d992534310d441210a2a8b31711c7e515c326a770d8b31f54a642d3f46R83